modules/tinymce/src/plugins/link/main/ts/ui/sections/LinkListOptions.ts

```diff
@@ -27,18 +26,22 @@ const getLinks = (editor: Editor): Promise<Optional<ListItem[]>> => {
   const extractor = (item: UserListItem) => editor.convertURL(item.value || item.url, 'href');
 
   const linkList = Options.getLinkList(editor);
-  return new Promise<Optional<UserListItem[]>>((callback) => {
+  return new Promise<Optional<UserListItem[]>>((resolve) => {
     // TODO - better handling of failure
     if (Type.isString(linkList)) {
-      XHR.send({
-        url: linkList,
-        success: (text) => callback(parseJson(text)),
-        error: (_) => callback(Optional.none())
-      });
+      window.fetch(linkList)
```

Comment:
Similar to before, lets add `fetch` to the allowed dom globals list instead of having to use `window.fetch` everywhere.